src/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+var _ = { extend: require('lodash.merge') };
+
+var rememberMe = module.exports = {
+  key: 'rememberMe'
+};
+
+rememberMe.set = function( params, errorCallback ) {
+  var defaults = {
+    url: location.href
```

Comment:
If we are trying to make this generic should we set this as a default rather than requiring the user to pass it in?